crates/autopilot/src/solvable_orders.rs

```diff
@@ -337,6 +340,72 @@ impl SolvableOrdersCache {
             .collect()
     }
 
+    /// Returns current solvable orders along with the latest order creation
+    /// timestamp.
+    async fn get_solvable_orders(&self) -> Result<(SolvableOrders, DateTime<Utc>)> {
```

Comment:
Is it possible to simplify this query a lot by always using `solvable_orders_after()`?
All you need for that is the list of known orders, latest timestamp and latest block.
If the cache is populated you just take those values and if it's not you should be able to just use defaults for everything, no?

Reply:
> Is it possible to simplify this query a lot by always using solvable_orders_after()?

`solvable_orders_after` doesn't filter orders in the SQL query, so it would return the whole table at the first start if the creation timestamp/block number contains default values. I tried to explain this in the comment:
https://github.com/cowprotocol/services/blob/1447bba2fe04c1f87d8eddd1819e8ac105eda295/crates/autopilot/src/solvable_orders.rs#L348-L358